Cargo.toml

```diff
@@ -11,13 +11,18 @@ categories = ["web-programming::http-server"]
 keywords = ["warp", "server", "http", "hyper"]
 autotests = true
 autoexamples = true
+edition = "2018"
+
+[package.metadata.docs.rs]
+features = ["tls"]
 
 [dependencies]
 bytes = "0.4.8"
-futures = "0.1"
+futures-preview = "0.3.0-alpha.18"
+futures-util-preview = "0.3.0-alpha.18"
 headers = "0.2"
 http = "0.1"
-hyper = "0.12.18"
+hyper = "=0.13.0-alpha.1"
```

Comment:
hyper alpha 2 is released. I think here we can upgrade this dependency and just add some modifications in server code to adapt to the new version https://github.com/zimond/warp/commit/139a5abafdc23ff169af566f54147a38486a1fc8

Reply:
thanks! had to also implement `Accept` due to `AddrIncoming` no longer implementing Stream